docs/content/latest/reference/configuration/yugabyted/errors-glossary.md

```diff
@@ -0,0 +1,65 @@
+---
+title: yugabyted Errors and Warnings
+linkTitle: Errors and Warnings
+description: yugabyted CLI errors and warnings
+section: REFERENCE
+menu:
+  latest:
+    identifier: yugabyted-errors-warnings
+    parent: configuration
```

Comment:
You've got this in a folder called yugabyted, but the parent is configuration. Is that on purpose, or did you want it to show up as a child of yugabyted in the left nav? (If so, we need to rework a tiny bit)

Reply:
Yes, in left nav panel I want this to show under yugabyted because this page lists errors and warnings from yugabyted CLI

Reply:
@polarweasel - need your help with this